src/components/global/Footer.jsx

```diff
@@ -13,20 +13,20 @@ import { FaDiscord, FaGithub, FaLinkedin, FaMeetup } from 'react-icons/fa6';
 
 // Legal links
 const legalLinks = [
+  // {
+  //   href: 'https://www.codepdx.org/',
+  //   title: 'About'
+  // },
   {
-    href: 'https://www.codepdx.org/',
-    title: 'About'
-  },
-  {
-    href: 'https://www.codepdx.org/',
+    href: 'https://github.com/codeforpdx/codeofconduct',
     title: 'Code of Conduct'
   },
   {
-    href: 'https://www.codepdx.org/',
+    href: 'https://docs.google.com/forms/d/e/1FAIpQLSfu4u9FB00RXd2TncbMLws_Ckot-vYqPK0oUHr81uLDjThE7Q/viewform',
     title: 'Contact Us'
```

Comment:
maybe we title this 'sign up' since we have 'contact us' in other places that trigger the contact model.

Reply:
I'll change that. good idea. I wasn't really sure what to do so I panicked and linked to the same page.